tools/gstf_preparation/gstf_preparation.py

```diff
@@ -92,6 +92,12 @@ def create_tables(conn):
         FROM transcript JOIN gene
         USING (gene_id)''')
 
+    cur.execute('''CREATE TABLE syntenic_region (
+        syntenic_region_name VARCHAR NOT NULL,
+        gene_id VARCHAR NOT NULL REFERENCES gene(gene_id),
+        species VARCHAR NOT NULL,
```

Comment:
I think this is a more logic order?
```suggestion
        species VARCHAR NOT NULL,
        syntenic_region_name VARCHAR NOT NULL,
        gene_id VARCHAR NOT NULL REFERENCES gene(gene_id),
```